### DISCUSSION TITLE
Annotation Panel: Show annotations from queries

### DISCUSSION THREAD

**Discussion by allomorphy** [grafana/grafana#39228] [Dashboards, Panels & Visualizations]
Thank you for this @ashharrison90 et al https://grafana.com/docs/grafana/latest/visualizations/annotations/?src=grafana_news&mdm=rss
It is quite inspiring and very well thought through.

I just spent a lot of time trying to have it work for our use case and I think that there is an issue that is not on my side.
Please excuse the brevity of this report because I am short of time now  😊 

I hope that this video is worth a thousand words 😉
![Grafana_Annotations_Panel_requireS_interactive_Annotation_Kapture](https://user-images.githubusercontent.com/50141922/133417035-15799d77-50ad-479f-9085-081e4d06b698.gif)

The Panel only picks up `manual` or interactive Annotations and not automatic Annotation based on queries.
So all those reddish `OGTT` queries on the timeline are automatic annotations set up with a query.
I need to go it and press the edit pencil and then save. Now Grafana's Annotation Panel sees them!

Surely unintended by the developer of this panel?

If more information is required, then I will be happy to supply.

Thanks again for this feature.
Best wishes
Eric

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
